x-pack/plugins/security_solution/public/common/components/header_page/index.tsx

```diff
@@ -117,45 +97,43 @@ const HeaderPageComponent: React.FC<HeaderPageProps> = ({
     [backOptions, history]
   );
   return (
-    <Header border={border} {...rest}>
-      <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
-        <FlexItem>
-          {backOptions && (
-            <LinkBack>
-              <LinkIcon
-                dataTestSubj={backOptions.dataTestSubj ?? 'link-back'}
-                onClick={goTo}
-                href={formatUrl(backOptions.href ?? '')}
-                iconType="arrowLeft"
-              >
-                {backOptions.text}
-              </LinkIcon>
-            </LinkBack>
-          )}
+    <EuiPageHeader alignItems="center" bottomBorder={border} paddingSize="l">
+      <EuiPageHeaderSection>
```

Comment:
And one last question regarding this current implementation. It seems like the title is always wrapped in a tooltip that is just a duplicate of the visible text. Is there a known reason for this?

<img width="166" alt="Screen Shot 2021-06-15 at 14 45 56 PM" src="https://user-images.githubusercontent.com/549577/122154910-f2608d80-ce33-11eb-82f0-32b1d7ffc655.png">

Reply:
Yea, maybe we can get on a call with @bfishel to figure out how we want to break some of these apart? Happy to do an updated `header` migration once we have the pages broken apart as part of our Phase 2?

Reply:
> And one last question regarding this current implementation. It seems like the title is always wrapped in a tooltip that is just a duplicate of the visible text. Is there a known reason for this?
> 
> <img alt="Screen Shot 2021-06-15 at 14 45 56 PM" width="166" src="https://user-images.githubusercontent.com/549577/122154910-f2608d80-ce33-11eb-82f0-32b1d7ffc655.png">

I was confused too, but I believe it's because we have a common header component, and some of our headers are things like host names and network names and those can be lengthy, and be forced into an ellipse. This accounts for those scenarios, but then ends up getting applied to all our headers.

Reply:
I'd probably just lean into the `title` attribute then on elements that get truncated. That way it's a long hover and uses the browser's internal tooltip so it's not always visible to users as they roll over the page title.